.github/workflows/ubuntu-syscollector-tests.yml

```diff
@@ -0,0 +1,29 @@
+name: Syscollector test on Ubuntu
+
+on:
+  pull_request:
+    branches: ["*"]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.11
```

Comment:
We should consistent with the other tests: here the `architecture` isn't defined.